### ISSUE TITLE
Warnings when including slides

### ISSUE THREAD

**Issue by pjforde1978** [Zulko/eagle.js#68]
My primary Slideshow was getting unwieldy, so I moved the individual slides into components. Functionally, it works great.

However, my debug console is now blowing up with 4 warnings for every included slide:

`[Vue warn]: Error in mounted hook: "TypeError: Cannot set property 'fontSize' of undefined" found in ---> <Intro> at src/slideshows/xxx/Intro.vue`

`TypeError: Cannot set property 'fontSize' of undefined`

`[Vue warn]: Error in callback for watcher "computedActive": "TypeError: Cannot set property 'visibility' of undefined" found in ---> <Intro> at src/slideshows/xxx/components/Intro.vue`

`TypeError: Cannot set property 'visibility' of undefined`

I am admittedly new to Vue but I'm enjoying learning it via EagleJS. It's possible that I'm missing something dumb.

In my main Slideshow:

`intro(:inserted='true')`

and Intro.vue:

```
<template lang="pug">
slide(enter='fadeIn' leave='fadeOut')
  h1 Test
</template>
<script>
import { Slideshow } from 'eagle.js'
export default {
  mixins: [Slideshow]
}
</script>
```

Like I said... everything seems to be working just fine. I just wish it wasn't throwing so many warnings.

**Comment by yaodingyd** [comment]
Can you use this link https://codesandbox.io/s/m5w17zvp39 to reproduce your issue please?

**Comment by pjforde1978** [comment]
@yaodingyd yes, I can: https://codesandbox.io/s/7wr3vv5k7x

**Comment by yaodingyd** [comment]
Any `slideshow` should always have a wrapper on `slide`. `slideshow` initialization expects there is at least one element. 

I will add this to readme, or if you like, a PR is welcome!

Thanks!

**Comment by pjforde1978** [comment]
@yaodingyd I'm so sorry - I don't understand your answer. I'm new to Eagle and to Vue.

If I knew how to fix this, I would. ;)

**Comment by yaodingyd** [comment]
No worry. I mean you can add a div before your slide in Example.vue

**Comment by pjforde1978** [comment]
@yaodingyd Of course, it would be so simple. Thank you very much - it works like a charm, now.
